package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ripple-binary-codec",
-  "version": "0.2.6",
+  "version": "0.3.0",
```

Comment:
Can we start a CHANGELOG? We do this for most Xpring projects. 

You can steal ours here: 
https://github.com/xpring-eng/Xpring-JS/blob/master/CHANGELOG.md

Reply:
We have a changelog already (but it's called HISTORY.md) - https://github.com/ripple/ripple-binary-codec/blob/master/HISTORY.md

It is ideal to update the changelog with every PR, and put the PR's changes under an "UNRELEASED" heading. However, we're not sticklers for this; checking and updating the changelog is part of our release process.